R/source.R

```diff
@@ -215,6 +207,49 @@ rust_function <- function(code, env = parent.frame(), ...) {
   rust_source(code = code, env = env, ...)
 }
 
+invoke_cargo <- function(
+  toolchain, specific_target,
+  dir, profile, stdout, stderr,
+  use_rtools) {
+  # Append rtools path to the end of PATH on Windows
+  if (
+    use_rtools &&
```

Comment:
Since this is a parameter a user can set, you should write `isTRUE(use_rtools)`.